diff --git a/pyproject.toml b/pyproject.toml
index d090d92..22fe386 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -30,7 +30,7 @@ keywords = ["msilib"]
 license = "PSF-2.0"
 license-files = ["LICENSE"]
 readme = "README.md"
-requires-python = ">=3.13"
+requires-python = ">=3.10"
 
 [[project.authors]]
 name = "Marcelo Duarte"
diff --git a/tests/test_msilib.py b/tests/test_msilib.py
index 45388d1..d3dd0d2 100644
--- a/tests/test_msilib.py
+++ b/tests/test_msilib.py
@@ -3,15 +3,15 @@
 from __future__ import annotations
 
 import os
+import sys
 from typing import TYPE_CHECKING
 
 import pytest
 
 if TYPE_CHECKING:
     from collections.abc import Generator
-    from pathlib import Path
-
     from msilib._msi import _Database
+    from pathlib import Path
 
 msilib = pytest.importorskip("msilib", reason="Windows tests")
 schema = pytest.importorskip("msilib.schema", reason="Windows tests")
@@ -108,6 +108,9 @@ def test_directory_start_component_keyfile(
         msilib._directories.clear()  # noqa: SLF001
 
 
+@pytest.mark.skipif(
+    sys.version_info[:2] < (3, 13), reason="Only for Python 3.13+"
+)
 def test_large_package(db: _Database, tmp_path: Path) -> None:
     root_dir = tmp_path / "root"
     root_dir.mkdir()
